test/e2e_env/kubernetes/gateway/gateway.go

```diff
@@ -120,13 +119,9 @@ type: system.kuma.io/secret
 
 		Eventually(func(g Gomega) {
 			var err error
-			clusterIP, err = k8s.RunKubectlAndGetOutputE(
-				kubernetes.Cluster.GetTesting(),
-				kubernetes.Cluster.GetKubectlOptions(namespace),
-				"get", "service", "simple-gateway", "-ojsonpath={.spec.clusterIP}",
-			)
+			gwIP, err = kubernetes.Cluster.GetServiceIP("simple-gateway", namespace)
```

Comment:
This might actually give the "wrong" IP, right? Since it might be a load balancer, and a load balancer also has a ClusterIP.

Reply:
True, you are right, maybe I can add a `type` so you can choose which address you want